packages/fxa-auth-server/lib/senders/emails/layouts/subscription/en.ftl

```diff
@@ -22,4 +23,6 @@ subplat-privacy-policy-plaintext = { subplat-privacy-policy }:
 subplat-cloud-terms = { -product-firefox-cloud } Terms of Service
 subplat-cloud-terms-plaintext = { subplat-cloud-terms }:
 subplat-legal = Legal
+subplat-legal-plaintext = { subplat-legal }:
```

Comment:
Not really for this issue, but wonder if we could have stripped out the `:` on these strings and resuse the value in the html version?

Reply:
FxA email template uses the same string values in both the HTML and plaintext versions for Legal and Privacy. I had added `:`s to strings to be consistent with the rest of the strings within the subplat email templates.

Reply:
I think we should leave them like we had them to reflect parity of the old templates, but, I think it's worth running by whomever we're going to run our [confirmation needed doc](https://docs.google.com/document/d/1dGwY1yOHk4X9yoXmg4sTidlzZ5c-wLUdTzMLiH7Pv4E/edit) too. 😬

Reply:
For consistency within subplat email templates, they will end with colons. It has been noted within our document.